dronegen/container_image_products.go

```diff
@@ -119,14 +138,22 @@ func NewTeleportOperatorProduct(cloneDirectory string) *Product {
 			case "armhf":
 			case "arm":
 				buildboxName += "-arm"
-				compilerName = "arm-linux-gnueabihf-gcc"
+				compilerName = "arm-linux-gnueabi-gcc"
```

Comment:
Why did we change this?

Reply:
You're right. This should not change. I fixed the packages and reverted the compiler name.